### PR TITLE
Use input with type=tel to automaticaly open numeric keyboard on mobile devices

### PR DESCRIPTION
**Pull request by alexeystadnikov-okta**
screenshot_20171218-181339
screen shot 2017-12-18 at 6 02 56 pm
screen shot 2017-12-29 at 4 06 42 pm

Resolves OKTA-79239

@mauriciocastillosilva-okta 
@eugenebakaiev-okta 
Please, review.

### CODE REVIEW

**Comment by mauriciocastillosilva-okta** [comment]
@alexeystadnikov-okta just to confirm, did you try this in iOs and in Android?

File: test/unit/helpers/util/Expect.js [hunk 1]
```diff
@@ -152,9 +152,10 @@ define([
     });
   };
 
-  fn.isTextField = function ($input) {
+  fn.isTextField = function ($input, inputType) {
```
Comment (eugenebakaiev-okta) [inline test/unit/helpers/util/Expect.js]: Maybe it would be better to rename this method, as it's purpose changed?

Comment (alexeystadnikov-okta) [reply test/unit/helpers/util/Expect.js]: Done. Added another method instead of modification the existing one.

**Comment by alexeystadnikov-okta** [comment]
@mauriciocastillosilva-okta 
Tried this on Android.
Don't have a possibility to test on iOS for the moment.

**Comment by alexeystadnikov-okta** [comment]
Checked on the iOS emulator. Works fine.

File: test/unit/spec/MfaVerify_spec.js [hunk 2]
```diff
@@ -2178,7 +2178,7 @@ function (Okta,
               var form = test.form[1];
               form.inlineTOTPAdd().click();
               expect(form.inlineTOTPAdd().length).toBe(0);
-              Expect.isTextField(form.answerField());
+              Expect.isTelField(form.answerField());
```
Comment (mauriciocastillosilva-okta) [inline test/unit/spec/MfaVerify_spec.js]: can we use the same function as in the other tests?  ```expectHasAnswerField```? That way we don't need to implement a new method in ```Expect``` that does the same thing.

Comment (alexeystadnikov-okta) [reply test/unit/spec/MfaVerify_spec.js]: Done

File: test/unit/helpers/util/Expect.js [hunk 3]
```diff
@@ -1,4 +1,4 @@
-/* eslint max-statements:[2, 28] */
+/* eslint max-statements:[2, 30] */
```
Comment (mauriciocastillosilva-okta) [inline test/unit/helpers/util/Expect.js]: can we revert this? It is probably not needed anymore

Comment (alexeystadnikov-okta) [reply test/unit/helpers/util/Expect.js]: Done

**Pull request merged**